### PR TITLE
Fix kura-kapua docs grammatical errors

### PR DESCRIPTION
**Pull request by ghost**
This PR fixes kura-kapua docs grammatical errors.

### CODE REVIEW

File: docs/kuraKapuaDocs.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ kapua-password
 
 ### Adding Account and User
 
-Now you can login into Kapua, but we still need to create an Account and a User. This section has for steps. First one is creating a child account. 
+Now you can login to Kapua, but we still need to create an Account and a User. This section has four steps. The first one is creating a child account.
```
Comment (LeoNerdoG) [inline docs/kuraKapuaDocs.md]: "The first one" is not gramatically correct. Please change it back to "First one...."

File: docs/kuraKapuaDocs.md [hunk 2]
```diff
@@ -144,10 +144,10 @@ After that go to **Permissions** tab and click **Add**, select **ALL** in every
 
 Now we have everything we need to connect Kura to Kapua.  
 
-## Downloading and cofiguring Kura
+## Downloading and configuring Kura
 
-Second part of this guide describes how to configure Raspberry Pi, but you can also use BBB (BeagleBone Black) or Intel Edison [as described on Kura download page](http://www.eclipse.org/kura/downloads.php?).
-Because there already exists guide for [installing Kura on Raspberry Pi](https://eclipse.github.io/kura/intro/raspberry-pi-quick-start.html) we will not repeat the procedure here. When you finish, come back and follow procedure bellow. In this example wi will assume that Raspberry Pi has IP: 192.168.1.11 and our PC 192.168.1.10.
+The second part of this guide describes how to configure Raspberry Pi, but you can also use BBB (BeagleBone Black) or Intel Edison [as described on Kura download page](http://www.eclipse.org/kura/downloads.php?).
```
Comment (LeoNerdoG) [inline docs/kuraKapuaDocs.md]: "The second part" is not gramatically correct. Please change it back to "Second part...."

File: docs/kuraKapuaDocs.md [hunk 3]
```diff
@@ -181,7 +181,7 @@ Click **+** right from **search** box. Under **Factory** select **org.eclipse.ku
 
 Example publisher should automatically start sending data to [Kapua](http://localhost:8080/), which can be verified in kura.log file or in Kapua itself (kura.log file is in on Raspberry Pi in /var/log folder).
 
-If everything done correctly data should be transmitted to Kapua and accumulated in **data/metrics**. 
+If everything is done correctly data should be transmitted to Kapua and accumulated in **data/metrics**.
```
Comment (LeoNerdoG) [inline docs/kuraKapuaDocs.md]: Missing comma between "correctly" and "data".

**Review by LeoNerdoG — CHANGES_REQUESTED** [review]
hey @geetha-rangaswamaiah,
I have checked your changes, and there are some fixes to be made. Please go through document one more time. Thanks!
I would also like to kindly ask you to pack all the syntax error fixes into one PR instead of creating tnes of PRs just for syntax errors...

**Review by LeoNerdoG — APPROVED** [review]

**Comment by LeoNerdoG** [comment]
Hey @Coduz,
I have checekd this, it is OK, you can merge this.

**Comment by Coduz** [comment]
Hi @geetha-rangaswamaiah  ,

This has been merged in `develop` and backported in `release-1.2.x` and released with Kapua `1.2.5`.

Regards,

Alberto

**Pull request merged**